pegasus/test/test_script_constants.rb

```diff
@@ -56,4 +56,15 @@ def test_category_index
     assert_nil ScriptConstants.position_in_category('script', :not_a_category)
     assert_nil ScriptConstants.position_in_category('not a script', :csp)
   end
+
+  describe 'ScriptConstants::script_in_any_category?' do
```

Comment:
Why do we want to move testing of `lib/cdo/script_constants.rb` from `lib/test/` to `pegasus/test/`?